service/eventer/spec/spec.go

```diff
@@ -5,18 +5,14 @@ type EventerType string
 
 // Eventer represents a Service that checks for deployment events.
 type Eventer interface {
-	// Boot initializes the eventer and makes it run.
+	// Boot initializes the eventer and makes it run. The eventer decides if the
+	// TPO it watches should be initialized and/or updated using
+	// NewDeploymentEvents.
 	Boot()
 	// NewDeploymentEvents returns a channel of DeploymentEvents. This channel can
 	// be ranged over to receive DeploymentEvents as they come in. In case of
 	// error during setup, the error will be non-nil.
 	NewDeploymentEvents() (<-chan DeploymentEvent, error)
-	// SetPending updates the DeploymentEvent remote state to a pending state.
-	SetPending(DeploymentEvent) error
-	// SetSuccess updates the DeploymentEvent remote state to a success state.
-	SetSuccess(DeploymentEvent) error
-	// SetFailed updates the DeploymentEvent remote state to a failed state.
-	SetFailed(DeploymentEvent) error
```

Comment:
This will be done by the operator. The eventer cannot know this because it just provides deployment event information. The reporting has to be done somewhere else.